pkg/store/consul/consulutil/watch.go

```diff
@@ -327,7 +328,7 @@ func WatchDiff(
 				mapCopy[key] = val
 			}
 
-			outgoingChanges := &WatchedChanges{}
+			outgoingChanges := &WatchedChanges{Total: len(pairs)}
```

Comment:
Well... it's a bit weird because the other PR only does this for `WatchDiff`, which means that the only affected farm is the DS farm. the RU farm doesn't use WatchDiff  - it uses a plain Watch (and is probably internally doing its own diff...)